app/main.py

```diff
@@ -1 +1,19 @@
 # write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, file, dir_file = command.split()
```

Comment:
file - is the bad name it this case, first of all, it is source_file_path
dir_file - also not a good name. I totally does not understand what it is mean) Use destination_file_path
Also validate command for 3 params
mv - it is not mv it input_command